content/onionservices/onionservices-5/contents.lr

```diff
@@ -7,15 +7,18 @@ seo_slug: address-bar-onion-icons
 description:
 When browsing an Onion Service, Tor Browser displays different onion icons in the address bar indicating the security of the current webpage.
 
-![Image of a green onion](/static/images/green-onion.png)
-A green onion means:
-- The Onion Service is served over HTTP, or HTTPS with a self-signed certificate.
+![Image of a black onion](/static/images/black-onion.png)
+A black onion means:
+- The Onion Service is served over HTTP, or HTTPS with a CA-Issued certificate.
+- The Onion Service is served over HTTPS with a Self-Signed certificate.
+- The Onion Service is served over HTTPS with EV.
```

Comment:
What's the difference of "- The Onion Service is served over HTTPS with EV." and "- The Onion Service is served over HTTP, or HTTPS with a CA-Issued certificate.". I think the latter also includes the former.

Reply:
Fixed!